Lib/dataclasses.py

```diff
@@ -174,10 +174,19 @@ def __init__(self, name):
         self.name = name
     def __repr__(self):
         return self.name
+    def __str__(self):
+        return _NAME_MAPPING[self]
+
 _FIELD = _FIELD_BASE('_FIELD')
 _FIELD_CLASSVAR = _FIELD_BASE('_FIELD_CLASSVAR')
 _FIELD_INITVAR = _FIELD_BASE('_FIELD_INITVAR')
 
+_NAME_MAPPING = {
+    _FIELD: 'Field',
+    _FIELD_CLASSVAR: 'ClassVar',
+    _FIELD_INITVAR: 'InitVar'
+}
+
```

Comment:
I don't think it's worth adding this mapping and the new `__str__` just for this one error message.